CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Twilio Voice React Native SDK has now reached milestone `beta.3`. Included in th
 ## Features
 
```

Comment:
Before publishing `beta.3`, we should probably note that these features are Android specific.

Reply:
done